lib/ansible/cli/galaxy.py

```diff
@@ -425,9 +437,24 @@ def add_install_options(self, parser, parents=None):
                                         help='Include pre-release versions. Semantic versioning pre-releases are ignored by default')
             install_parser.add_argument('-U', '--upgrade', dest='upgrade', action='store_true', default=False,
                                         help='Upgrade installed collection artifacts. This will also update dependencies unless --no-deps is provided')
+            install_parser.add_argument('--keyring', dest='keyring', default='~/.ansible/pubring.kbx',
+                                        help='The keyring used during signature verification')
+            install_parser.add_argument('--disable-gpg-verify', dest='disable_gpg_verify', action='store_true',
+                                        help='Disable GPG signature verification when installing collections from a Galaxy server')
```

Comment:
Can we add config support for this too?

Reply:
I added a config entry to use as the default:
```yaml
GALAXY_DISABLE_GPG_VERIFY:
  default: false
  type: bool
  env:
  - name: ANSIBLE_GALAXY_DISABLE_GPG_VERIFY
  ini:
  - section: galaxy
    key: disable_gpg_verify
  description:
  - Disable GPG signature verification during collection installation.
```